lab_1_classify_by_unigrams/main.py

```diff
@@ -88,6 +166,17 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
     :paths_to_profiles: a list of strings to the profiles
     :return: a list of loaded profiles
     """
+    if not isinstance(paths_to_profiles, list):
+        return None
+    list_processed_profiles = []
+    for paths in paths_to_profiles:
+        if isinstance(paths, str):
+            language_profile = load_profile(paths)
+            if isinstance(language_profile, dict):
+                processed_profile = preprocess_profile(language_profile)
+                if isinstance(processed_profile, dict):
```

Comment:
so many tabs... do you really need it?

Reply:
There was a "spell check" error without these tabs